cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -211,42 +194,58 @@ private boolean isUserDataset(DatasetSpecificationSummary specification) {
     return !dsNamespace.contains(specification.getName(), Constants.SYSTEM_NAMESPACE);
   }
 
-  private void assertNoUserDatasets() throws Exception {
+  private void truncateAllStreams(Id.Namespace namespace) throws IOException, UnauthorizedException {
+    // TODO: check no streams once streams can be deleted instead of truncating all streams
```

Comment:
Streams can be deleted now.
https://github.com/caskdata/cdap/blob/feature/pfs-metadata/cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHandler.java#L263-263